finagle-memcachedx/src/main/scala/com/twitter/finagle/memcachedx/protocol/text/client/DecodingToResponse.scala

```diff
@@ -73,6 +73,6 @@ class DecodingToResponse extends AbstractDecodingToResponse[Response] {
   }
 
   private[this] def parseErrorMessage(tokens: Seq[Buf]) =
-    tokens.lastOption map { case Buf.Utf8(s) => s } getOrElse("")
+    if (tokens.isEmpty) "" else tokens.tail map { case Buf.Utf8(s) => s  } mkString(" ")
```

Comment:
And why the empty `tokens` considered as a special case? `Nil.mkString(" ")` will produce a "" string as expected.